ext/afform/mock/tests/phpunit/api/v4/AfformUsageTest.php

```diff
@@ -102,8 +102,8 @@ public function testAboutMeForbidden() {
         ->execute();
       $this->fail('Expected authorization exception from Afform.submit');
     }
-    catch (\Civi\API\Exception\UnauthorizedException $e) {
-      $this->assertRegExp(';Authorization failed: Cannot process form mock\d+;', $e->getMessage());
+    catch (\API\_Exception $e) {
```

Comment:
is right @colemanw ? surely it needs to be \API_Exception because isn't the class API_Exception?

Reply:
Whoops. Good catch. Not as funny a typo than eileen's lasers though...